vendors/vendor/boards/board/aws_demos/config_files/ota_config.h

```diff
@@ -0,0 +1,207 @@
+/*
+ * AWS IoT Device SDK for Embedded C V202012.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file ota_config.h
+ * @brief OTA user configurable settings.
+ */
+
+#ifndef OTA_CONFIG_H_
+#define OTA_CONFIG_H_
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Configure name and log level for the OTA library. */
+#ifndef LIBRARY_LOG_NAME
+    #define LIBRARY_LOG_NAME     "OTA"
+#endif
+#ifndef LIBRARY_LOG_LEVEL
+    #define LIBRARY_LOG_LEVEL    LOG_ERROR
+#endif
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/* Include config file used for testing the OTA PAL. This config file defines
+ * the otapalconfigCODE_SIGNING_CERTIFICATE macro. */
+#include "ota_demo_config.h"
+
+/**
+ * @brief Log base 2 of the size of the file data block message (excluding the
+ * header).
+ *
+ * <b>Possible values:</b> Any unsigned 32 integer. <br>
+ */
+#define otaconfigLOG2_FILE_BLOCK_SIZE           12UL /* FIX ME. */
```

Comment:
Do these FIXMEs need to be removed? If not, should these be added to the same file in the tests folder?

Reply:
These were left here intentionally to show that they need to be updated by the user. This is in comparison to a configuration that could be changed, but could probably remain as the default. In this case these are very platform specific and should be updated per platform.

In the case of the test folder, they are required for compiling but the values are arbitrary. So they can remain the same without being modified.

Reply:
I would prefer TODO by user instead of FIXME.  FIXME implies a bug that we need to fixup.  TODO is just work that is to be done and some IDE's will even make a list of these items.

Reply:
Done.